src/namedMessages/namedMessages.mjs

```diff
@@ -0,0 +1,20 @@
+const messages = {
+  BLOCK_SEAT_SUCCESS: { success: false, message: 'seat is already blocked' },
```

Comment:
you can out here just strings, but the idea is right

Reply:
fixed